.circleci/config.yml

```diff
@@ -3,16 +3,59 @@
 # Check https://circleci.com/docs/2.0/language-ruby/ for more details
 #
 version: 2.1
+
+executors:
+  ruby_browsers:
+    docker:
+      - image: cimg/ruby:3.0.4-browsers
+        environment:
+          BUNDLER_VERSION: 2.3.13
+
+commands:
+  bundle-yarn-install:
```

Comment:
[suggestion] i'm not super familiar with how circleci works, but it looks like a lot of this code is currently replicated in the build/deploy steps. that might be necessary, but i am wondering if you could just pull out the `Install dependencies` step and add the yarn install step?

Reply:
By making this a command, it reduces duplication. Otherwise it'd all have to be spelled out in the `build` and `deploy` steps. There is a little duplication with the bundle install in the yarn steps, but I'm not 100% sure if that needs to be treated special outside of the normal bundle install. Looking at [CircleCI](https://app.circleci.com/pipelines/github/18F/identity-saml-sinatra/178/workflows/c7e15a11-c211-4d2d-9e70-2a503680e093/jobs/537) it looks like in the `Install Yarn` steps it doesn't do the full `bundle install` for the project there.

Reply:
Or maybe it is duplicated effort looking closer. Hmm, I'll pull it out of the jobs and let it just live in the command.